src/Stripe.net/Services/Products/StripeProducNestedCreateOptions.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeProductNestedCreateOptions : INestedOptions
```

Comment:
Since it's nested under the plan options I think we might want to name it `StripePlanProductCreatOptions` to indicate the relation.

Not a huge deal though as naming is at best inconsistent in the library today already.

Reply:
My thought was there is nothing specific about the nesting to plans -- though you're right, that's the only place it's used and probably the only place it'll ever be used.